scripts/Tools/check_case

```diff
@@ -1,65 +1,30 @@
-#!/usr/bin/env perl
+#!/usr/bin/env python2
 
-use strict;
-use Cwd;
-use English;
-use Getopt::Long;
-use File::Basename;
-use IO::File;
-use IO::Handle;
-use XML::LibXML;
-sub usage {
-    die <<'EOF';
+"""
+Checks that case is set up correctly
+"""
 
-SYNOPSIS
+from standard_script_setup import *
 
-  check_case [options]
+from CIME.utils import expect, get_model, run_cmd
+from CIME.case  import Case
 
-  checks that case is set up correctly
+logger = logging.getLogger(__name__)
 
-OPTIONS
+parser = argparse.ArgumentParser()
```

Comment:
I'd prefer if this tool followed the convention of our other tools, even if check_case is pretty simple. I will refer you to scripts/Tools/check_input_data as a simple example of what I think we want every tool to look like. The exe will have a parse_command_line function and a _main_func that will make a call to a library to do the actual work.